themes/default/content/blog/project-scoped-stacks-in-self-managed-backend/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Project-Scoped Stacks in Self-Managed State Backends"
```

Comment:
I don't have an obvious suggestion here, but the title feels too technical to be relatable to most end-users. Even advanced ones. Maybe "Better Stack References in Self-Managed State Backends"? (Because that's what makes your use of the product better.)

Reply:
"Project support in Self-Managed State Backends"?